go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/btcsuite/btcd/btcutil v1.1.5
 	github.com/cometbft/cometbft-db v0.11.0
 	github.com/cometbft/cometbft-load-test v0.1.0
-	github.com/cometbft/cometbft/api v1.0.0-alpha.1
+	github.com/cometbft/cometbft/api v0.0.0-20240311183503-2f0dc67ee8d5
```

Comment:
why v0?
I think it should signal that it is v1 related. can we remove this tag later, from the registry?

Reply:
It's the format added by go itself, when using `go get ...`